src/commands/utility/send-message.js

```diff
@@ -0,0 +1,24 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { createSendMessageModal } = require('../../modals/send-message-modal');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('send-message')
+    .setDescription('Replies with your input!')
+    .addChannelOption((option) =>
+      option.setName('channel').setDescription('The channel to message into')
+    ),
+  async execute(interaction) {
+    const channel = interaction.options.getChannel('channel');
+    const channelId = channel?.id;
```

Comment:
What would happen if channel id is not found? not sure if we can add a default channel?

Reply:
The channel came form the slash command. If the user pass the channel the logic enter in the first conditional in the other hand the message will send to the same channel that open it